src/users/serializers.py

```diff
@@ -32,39 +32,40 @@ class UserSignUpSerializer(serializers.Serializer):
     confirm_password = serializers.CharField(
         write_only=True,
         required=True,
-        label="Confirm Password",
-        style={'input_type': 'password'}
+        label="Confirm Password", 
+        style={'input_type': 'password'} 
     )
 
     class Meta:
         model = User
         fields = ['username', 'email', 'password', 'confirm_password']
 
+    def validate_email(self, email):
```

Comment:
Ao invés de escrcever esse método, mudem o campo do email acrescentando o unique=True
```suggestion
   
    email = serializers.EmailField(
        required=True,
        unique=True,
        label="Email Address",
    )
```